staker-rewards/lib/rewards-calculator.js

```diff
@@ -98,6 +107,45 @@ export default class RewardsCalculator {
     return new BigNumber(minimumStake)
   }
 
+  checkRequirementsViolations(operatorParameters) {
+    const violations = []
+    const { isFraudulent, requirements, operatorSLA } = operatorParameters
+
+    if (isFraudulent === true) {
+      violations.push("isFraudulent")
+    }
+
+    if (requirements.factoryAuthorizedAtStart === false) {
+      violations.push("factoryAuthorizedAtStart")
+    }
+
+    if (requirements.poolAuthorizedAtStart === false) {
+      violations.push("poolAuthorizedAtStart")
+    }
+
+    if (requirements.poolDeauthorizedInInterval === true) {
+      violations.push("poolDeauthorizedInInterval")
+    }
+
+    if (requirements.minimumStakeAtStart === false) {
+      violations.push("minimumStakeAtStart")
+    }
+
+    if (requirements.minimumUnbondedValueRegisteredAtStart === false) {
+      violations.push("minimumUnbondedValueRegisteredAtStart")
+    }
+
+    if (operatorSLA.keygenSLA !== "N/A" && operatorSLA.keygenSLA < 90) {
+      violations.push("keygenSLA")
+    }
+
+    if (operatorSLA.signatureSLA !== "N/A" && operatorSLA.signatureSLA < 95) {
+      violations.push("signatureSLA")
+    }
```

Comment:
#638 introduced `poolRequirementFulfilledAtStart` requirement, let's make sure it's also covered.